src/content/docs/release-notes/org-user-mgmt-release-notes/org-users-21-09-01.mdx

```diff
@@ -0,0 +1,16 @@
+---
+subject: Organization and user management
+releaseDate: '2021-09-01'
+version: '210901'
```

Comment:
Just wanted to make sure this is the correct version number.

Reply:
yeah it doesn't really have a version number so i'm just using the date, which is how logs does it too.